pkg/koordlet/resmanager/cpu_suppress.go

```diff
@@ -158,16 +159,9 @@ func (r *CPUSuppress) calculateBESuppressCPU(node *corev1.Node, nodeMetric *metr
 }
 
 // calculateBESuppressPolicy calculates the be cpu suppress policy with cpuset cpus number and node cpu info
-func calculateBESuppressCPUSetPolicy(cpusetQuantity *resource.Quantity, oldCPUSetNum int, nodeCPUInfo *metriccache.NodeCPUInfo) []int32 {
-	// set the number of cpuset cpus no less than 2
-	cpus := int32(math.Ceil(float64(cpusetQuantity.MilliValue()) / 1000))
-	if cpus < 2 {
-		cpus = 2
-	}
-
+func calculateBESuppressCPUSetPolicy(cpus int32, oldCPUSetNum int, processorInfos []util.ProcessorInfo) []int32 {
```

Comment:
split the processors into several partitions(socket+node), try to get 1 core(2 processors) from each partition for each iteration